src/components/molecules/definitionitem/DefinitionItem.jsx

```diff
@@ -23,14 +49,21 @@ const DefinitionItem = ({ payload }) => {
 			)}
 			<div className="term-definition">
 				<div className="definition">{definitionText}</div>
-				{payload.aliases && <SynonymTable aliases={payload.aliases} />}
+				{nciConceptId && renderSupplementalLinks(nciConceptId)}
+				{aliases && <SynonymTable aliases={aliases} />}
 			</div>
 		</>
 	);
 };
 
 DefinitionItem.propTypes = {
 	payload: PropTypes.object,
+	drugInfoSummaryLink: PropTypes.object,
```

Comment:
when I said to add drugInfoSummaryLink and nciConceptId as props, I didn't think everything else needed to be moved to props as well.  Maybe we could have defined a shape for payload, where things like name and termID were required.